test/units/modules/cloud/amazon/test_elb_application_lb.py

```diff
@@ -0,0 +1,166 @@
+#
+# (c) 2017 Michael Tinning
```

Comment:
Copyright guidelines have changed recently to be much shorter (and easier to follow) [here](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright)

So you can delete the whole license snippet below this and add:

```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```